service/integration/keymanagement_test.go

```diff
@@ -54,17 +52,20 @@ func (s *KeyManagementSuite) TearDownSuite() {
 
 func (s *KeyManagementSuite) Test_CreateProviderConfig_NoMetada_Succeeds() {
```

Comment:
Things that happen every test can be placed in `s.SetupTest`, e.g.:

```suggestion
func (s *KeyManagementSuite) SetupTest() {
	testProvider := s.getUniqueProviderName("test-provider")
}

func (s *KeyManagementSuite) Test_CreateProviderConfig_NoMetada_Succeeds() {
```